html-tables/index.html

```diff
@@ -10,11 +10,11 @@
     <h1>Student Grade Table</h1>
     <table>
       <tbody>
-        <tr>
-          <td><strong>Student Name</strong></td>
-          <td><strong>Course</strong></td>
-          <td><strong>Grade</strong></td>
-        </tr>
+        <thead>
+          <th><strong>Student Name</strong></th>
+          <th><strong>Course</strong></th>
+          <th><strong>Grade</strong></th>
+        </thead>
```

Comment:
The `thead` should not be a child of the `tbody`. The `tbody` should encapsulate all `tr` elements that are not the header for the table.

Reply:
`thead` and `tbody` will be siblings.

Reply:
Thank you once again @codingcodymiller! Here is the updated code. I ensured the tbody is encapsulating all tr elements outside of the header for the table.
<img width="486" alt="Screen Shot 2021-12-01 at 11 51 46 AM" src="https://user-images.githubusercontent.com/76764048/144303946-1e534672-7164-46b9-bf94-8eac43d1fd02.png">
.